storage-service/src/main/java/io/quarkus/qe/consumers/UpdateRepositoryRequestConsumer.java

```diff
@@ -19,6 +19,10 @@
 import io.quarkus.qe.model.Repository;
 import io.smallrye.reactive.messaging.annotations.Blocking;
 
+import java.util.Optional;
+
+import static io.quarkus.qe.model.Repository.UNDEFINED_VALUE;
```

Comment:
My formatter moves the static import to the top.

Reply:
not mine :(